extensions/btcpaycheckout/src/Checkout.jsx

```diff
@@ -13,16 +14,19 @@ export default reactExtension(
 );
 
 function Extension() {
+  const options = useSelectedPaymentOptions();
   const { shop, checkoutToken } = useApi();
+  const hasManualPayment = options.some((option) => option.type.toLowerCase() === 'manualpayment');
   const appUrl = `PLUGIN_URL/checkout?checkout_token=${checkoutToken.current}`;
+
+  if (!hasManualPayment) return null;
+
   return (
-    <>
```

Comment:
are those empty tags needed or can they be deleted like you did?

Reply:
If I properly built the docker image and deployed it, it seems to work :+1:

Reply:
maybe we should do some console.log with some release number so we can see which version runs, also for debugging for merchants?

Reply:
NAh.. the empty tags can be deleted. 



> maybe we should do some console.log with some release number so we can see which version runs, also for debugging for merchants?

So we'd have the console in prod always?

Reply:
Yes not ideal but would it hurt? Not sure, would help or we can just tell people to deploy latest app to be sure it's not due to outdated app release.